pootle/apps/pootle_fs/management/commands/fs_commands/info.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from pootle_fs.management.commands import ProjectSubCommand
+
+
+class ProjectInfoCommand(ProjectSubCommand):
+    help = "List FS translations files managed by Pootle."
+
+    def handle(self, *args, **kwargs):
+        fs = self.get_fs(kwargs['project'])
+        self.stdout.write("Project: %s" % fs.project.code)
+        self.stdout.write("type: %s" % fs.fs_type)
+        self.stdout.write("URL: %s" % fs.fs_url)
```

Comment:
yep, the original `help` talked about listing translation files which was wrong - ive updated help